docs/docs/notification-center/react/react-components.md

```diff
@@ -543,3 +545,4 @@ If you want to hide it, you can use the prop `showUserPreferences` on the `Popov
 :::note
 Facing issues in using notification center? Check out FAQs [here](../FAQ)
 :::
+
```

Comment:
@Toms-Uluks Please remove this extra line
lint for docs is failing due to this!